pkg/app/echo-v2.go

```diff
@@ -40,15 +39,59 @@ func (a *echoV2) Namespace() string {
 
 func (a *echoV2) Install(t test.TestHelper) {
 	t.T().Helper()
-	oc.ApplyFile(t, a.ns, examples.EchoV2YamlFile())
+	oc.ApplyTemplate(t, a.ns, tcpEchoV2Template, nil)
 }
 
 func (a *echoV2) Uninstall(t test.TestHelper) {
 	t.T().Helper()
-	oc.DeleteFile(t, a.ns, examples.EchoV2YamlFile())
+	oc.DeleteFromTemplate(t, a.ns, tcpEchoV2Template, nil)
 }
 
 func (a *echoV2) WaitReady(t test.TestHelper) {
 	t.T().Helper()
 	oc.WaitDeploymentRolloutComplete(t, a.ns, "tcp-echo-v2")
 }
+
+const tcpEchoV2Template = `
+apiVersion: v1
+kind: Service
+metadata:
+  name: tcp-echo
+  labels:
+    app: tcp-echo
+spec:
+  ports:
+  - name: tcp
+    port: 9000
+  #- name: tcp-other
+  #  port: 9001
+  ## Port 9002 is omitted intentionally for testing the pass through filter chain.
```

Comment:
Cannot we remove these commented properties?

Reply:
I didn't want to remove this, as I wasn't exactly sure why this is commented-out. I wanted to keep the comments so that we notice it in the future and figure out why. Eventually, we'll probably consolidate `echo-v1` and `echo-v2` with `echo`, but we'll first need to understand why this port was removed.